packages/flutter_driver/lib/src/driver.dart

```diff
@@ -251,6 +266,14 @@ class FlutterDriver {
     return response['response'];
   }
 
+  void _logCommunication(String message)  {
+    if (_printCommunication)
+      _log.info(message);
+    f.File file = fs.file(p.join(testOutputsDirectory, 'flutter_driver_commands.log'));
```

Comment:
If you run multiple flutter_driver instances, you should provide them with different testOutputsDirectory. Otherwise, the output in there is meaningless anyways.

Added an option to disable logging to file, but I'll leave it turned on by default.